### ISSUE TITLE
Change position of Repeat Event page title

### ISSUE THREAD

**Issue by danieldegroot2** [thomersch/openstreetmap-calendar#98]
For the page title "New Event" on the Repeat Event page, add an offset of a few pixels or position (i.e. above existing content) ([code](https://github.com/thomersch/openstreetmap-calendar/blob/e179927b1008c876f9e24e2ecb2899fc43c2939e/osmcal/views.py#L383)), and/or add it as the (sub)name of the browser tab.

Sidenote, this page title is not displayed when simply adding an event, which is inconsistent. Is this intended? I could understand this as being to indicate you are creating a new event instead of editing the existing one.

![image](https://user-images.githubusercontent.com/67521919/157752976-5c961401-d761-41ed-9e24-100d230e02ca.png)

**Comment by thomersch** [comment]
Yes, they are not very consistent. I think this is a result of some stylesheet changes that I made but haven't check on all the pages if they're still ok.

The general idea is that almost all pages have narrow, center content, because it's easier to read, but there are a few which use a multi-column design (like the event detail pages) and fallback to single column on narrow devices. The titles themselves should be offset as you say as on e.g. the documentation page.
